### DISCUSSION TITLE
Drop Down for Tenant Selection

### DISCUSSION THREAD

**Discussion by 274188A** [abpframework/abp#7566] [Q&A] [answered]
Has anyone thought about implementing drop-down (select) for the login page's Tenant selection?

I can see it would be useful (user friendly) but perhaps there's security issues that make this a bad idea...

Thoughts anyone?

**Comment by gterdem** [comment, marked as answer]
It sounds practical at first glance however it is a security (and business) risk even for a tenant to know the number of total tenants. 

However your institution may not be related with any of these issues so you may internally want to serve application with dropdown with a few tweaks to login page.
